src/xr/xr-manager.js

```diff
@@ -37,11 +39,12 @@ function XrManager(app) {
 
     this._supported = !! navigator.xr;
 
-    this._available = {
-        XRTYPE_AR: false,
-        XRTYPE_VR: false,
-        XRTYPE_INLINE: false
-    };
+    this._available = { };
+
+    // Add all the supported session types
+    this._available[XRTYPE_INLINE] = false;
+    this._available[XRTYPE_VR] = false;
+    this._available[XRTYPE_AR] = false;
```

Comment:
What's this change for?

Reply:
The previous code:
```
    this._available = {
        XRTYPE_AR: false,
        XRTYPE_VR: false,
        XRTYPE_INLINE: false
    };
```
Had named the keys as 'XRTYPE_AR', 'XRTYPE_VR' and 'XRTYPE_INLINE' rather than the string values that are in the constants.js file.

This change uses the string values of the constants instead :)

Reply:
Ah interesting. Fair enough.